README.md

```diff
@@ -130,11 +131,36 @@ const Toggle = () => {
  export default Toggle;
 }
 ```
+### A Component accessing multiple global state values
+Accessing multiple state values is pretty straight forward, declare constants to hold different state values and access state values by calling `useRhinoValue("key")` passing a key identifying with a state value as an argument.
+
+`details.js`
+
+```jsx
+import { useRhinoValue } from "./states.js"
+
+const Datails= () => {
+  /*  
+      Seperate calls to useSetRhinoState return seperate state values
+      based on keys that identify with state value.
+  */
+  const userName = useRhinoValue("name")
+  const addressDetails = useRhinoValue("addressDetails")
+  return(<div>
+      <span>{userName}</span>
+  </div>
+  )
+
+}
+```
+
+## Online Playground
```

Comment:
I think `Online Playground` deserves a better position at the top of the document